solidity-v1/dashboard/src/reducers/threshold-authorization.js

```diff
@@ -105,6 +112,38 @@ const updateThresholdAuthData = (authData, { operatorAddress }) => {
   return updatedOperators
 }
 
+const addStakeToAuthData = (
+  authData,
+  {
+    owner,
+    authorizerAddress,
+    beneficiary: beneficiaryAddress,
+    operatorAddress,
+    amount: stakeAmount,
+    isFromGrant,
+    operatorContractAddress,
+  }
+) => {
+  const newStake = {
+    owner,
+    authorizerAddress,
+    beneficiaryAddress,
+    operatorAddress,
+    stakeAmount,
+    contract: {
+      contractName: AUTH_CONTRACTS_LABEL.THRESHOLD_TOKEN_STAKING,
+      operatorContractAddress: operatorContractAddress,
+      isAuthorized: false,
+    },
+    isStakedToT: false,
+    isFromGrant: isFromGrant ? isFromGrant : false,
+    canBeMovedToT: false,
```

Comment:
The stake created from liquid tokens can be moved to T network. So we should `canBeMovedToT: isFromGrant`.

Reply:
You are 💯 right! 🦅 👁️